src/app/+data-and-analytics/academic-age-group/academic-age-group.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="container mb-1">
+  <div class="text-center h2">{{organization.name}}</div>
+  <div class="row">
+    <div class="col-xs-6">
+      <div class="card shadow bg-white rounded m-1">
+        <div class="card-body">
+          <div class="card-text">
+            <div class="text-center h2">{{mean | async}}</div>
+            <div class="text-center">Average academic age of affiliated researchers 🛈</div>
+          </div>
+        </div>
+      </div>
+      <div class="card shadow bg-white rounded m-1">
+        <div class="card-body">
+          <div class="h5">Researchers and publications per academic age group</div>
+          <scholars-barplot id="academicAgeBarplot" scale="1" [input]="academicAge" maxOverride="{{maxOverride | async}}"></scholars-barplot>
+        </div>
+      </div>
+    </div>
+    <div class="col-xs-6">
+      <div class="card shadow bg-white rounded m-1">
+        <div class="card-body">
+          <div class="card-text">
+            <div class="text-center h2">{{median | async}}</div>
+            <div class="text-center">Median academic age of affiliated researchers 🛈</div>
```

Comment:
Are these characters `🛈` supposed to be there?

Reply:
As per mockup, yes. But this text should be from i18n. Will update.